src/spinegeneric/cli/check_data_consistency.py

```diff
@@ -37,29 +37,33 @@ def get_parser():
     return parser
 
 
+Fieldname = str
+Row = dict[Fieldname, str]
```

Comment:
Huh! This is a typing pattern I hadn't seen before. I like this a lot? A neat way to add more contextual information to objects represented by stdlib types, without needing to make a completely new class for typing purposes.

Reply:
Yep! Using this kind of [type alias](https://docs.python.org/3/library/typing.html#type-aliases) is even more lightweight than using a [`NewType`](https://docs.python.org/3/library/typing.html#newtype). It can serve as both shorthand and documentation of intent. (And since we're not using any type checkers, our only use of type hints is for documentation.)